packages/rancher-istio/charts/1.6.500/templates/admin-role.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+  namespace: istio-system
+  name: istio-admin
+rules:
+  - apiGroups:
+      - config.istio.io
+    resources:
+      - adapters
+      - attributemanifests
+      - handlers
+      - httpapispecbindings
+      - httpapispecs
+      - instances
+      - quotaspecbindings
+      - quotaspecs
+      - rules
+      - templates
+    verbs: ["get", "watch", "list"]
```

Comment:
Who does get to update this if not admin?

Reply:
The cluster-admin would.